evap/contributor/forms.py

```diff
@@ -13,7 +14,7 @@
 
 
 class CourseForm(forms.ModelForm, BootstrapMixin):
```

Comment:
please do the equivalent changes also in the staff edit form. obsolete questions must be shown in the list if they are in use-